tests/govtool-frontend/playwright/tests/2-delegation/delegation.loggedin.spec.ts

```diff
@@ -71,3 +74,55 @@ test("2X_2. Should include info button and voting power on the Signal-No-Confide
 
   await expect(dRepDirectoryPage.signalNoConfidenceInfoButton).toBeVisible();
 });
+
+test.describe("DRep dependent tests", () => {
+  let dRepGivenName = "test";
+  let dRepId = "drep1ef7uslcjhjqrn4vv2y39c3yn345gzjsg7yufn76zye3v6fkz23q";
```

Comment:
shouldn't this be based on network?

Reply:
@mesudip , This is an example scenario when no active dReps are in the list. The dRep ID and dRep name are being overwritten by the values from fetchFirstActiveDRepDetails. Should I update this behavior based on the network? What happens if dbsync is cleared on the testnet, and that specific dRep is no longer available in the dRep list?